lib/board.rb

```diff
@@ -61,7 +61,14 @@ def valid_coordinates_suite(ship)
 
   def valid_placement?(ship, coordinates)
     @coordinates = coordinates
-    valid_coordinates_suite(ship) && ship_overlap(ship, coordinates)
```

Comment:
it's not here (comment wouldn't let me put it in the right spot because that wasn't a change. But do we need to do line 57 comment?